modules/virt-vm-serial-console-web.adoc

```diff
@@ -12,6 +12,8 @@ tab in the *Virtual Machine Overview* screen of the web console.
 
 . In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
 . Click the *Virtual Machines* tab.
-. Select a virtual machine to open the *Virtual Machine Overview* screen.
+. Select a virtual machine to open the *Virtual Machine Overview* page.
 . Click *Console*. The VNC console opens by default.
 . Click the *VNC Console* drop-down list and select *Serial Console*.
+. Optional: Open the serial console in a separate window by clicking *Open Console in New Window*.
```

Comment:
There is an issue here, select `Serial Console` and click `Open Console in New Window`, it still opens VNC console. Users need to select serial console on the new window.
I've opened a ticket for it https://issues.redhat.com/browse/CNV-6150.

Reply:
Thanks for opening that ticket. @jelkosz do you think this can be fixed before 2.5 is released? If not, we should include a known issue in the release notes.